Kickstarter-iOS/Views/Controllers/PledgePaymentMethodsViewController.swift

```diff
@@ -178,6 +193,16 @@ final class PledgePaymentMethodsViewController: UIViewController {
   }
 }
 
+extension PledgePaymentMethodsViewController: PledgeCreditCardViewDelegate {
+  func didSelectCard(_ cardView: PledgeCreditCardView) {
+    if let cards: [PledgeCreditCardView] = self.viewModel.outputs.savedCards() as? [PledgeCreditCardView] {
```

Comment:
Instead of passing these views through the VM, let's keep references to them in an array in the VC. We can add them to that array when we create the views. That was we can just iterate over the array when selection changes and inform the view of the selected card. If that view is configured with that card then it can update its selected state.